lib/ImageHelper.php

```diff
@@ -365,7 +365,12 @@ public static function analyze_url( $url ) {
 				$tmp = str_replace(WP_CONTENT_DIR, '', $tmp);
 			}
 		} else {
-			if ( !$result['absolute'] ) {
+			// if upload dir does not contain site_url, the content-directory seems to be outside of the site_url
+			// therefore using site_url() would lead to a wrong content/ path
+			if (false === strpos($upload_dir['baseurl'], site_url())) {
+				// use HOME_URL and relative image path
+				$tmp = get_home_url().$tmp;
```

Comment:
This doesn't actually use a relative path. It just concatenates causing `http://domain.comhttp://domain.com/app/uploads/`